libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MatrixArea/MatrixArea.tsx

```diff
@@ -658,11 +657,6 @@ export class MatrixArea extends React.PureComponent<
   }
 
   private textColorForBackground(colorStr: string): string {
-    return this.isColorDark(colorStr) ? "white" : "rgba(0,0,0,0.8)";
-  }
-
-  private isColorDark(colorStr: string): boolean {
-    const val = Lab(colorStr).l;
-    return val <= 65;
+    return isColorDark(colorStr) ? "white" : "rgba(0,0,0,0.8)";
```

Comment:
It would be nice if we didn't hard code colors because this will come back to bite us when supporting dark mode. Just saying... I don't particularly care if you address it now or later, but this will likely come up again.

Reply:
good point, will move it to constants in next PR